server/src/routes/course/taskVerification.ts

```diff
@@ -88,6 +88,13 @@ export const createTaskVerification = (_: ILogger) => async (ctx: Router.RouterC
   }
 };
 
+function isNextSubmitAllowed(hours: number, lastAttemptTime?: string) {
+  if (!hours || !lastAttemptTime) return true;
+  const diff = new Date(lastAttemptTime).getTime() - new Date().getTime() + hours * 60 * 60 * 1000;
```

Comment:
since we already use moment here, maybe try to make it more readable?

```typescript
const diff = moment(lastAttemptTime).subtract(hours, 'hour').diff(moment())
```